Extreme_Motorcycle_Class/src/emc/Utils.java

```diff
@@ -10,4 +15,69 @@ public static LocalDate createLocalDate(int year, int month, int day) throws Dat
         }
         return LocalDate.of(year, month, day);
     }
+
+    public static void printSet(ResultSet r) {
+        // Used later to get metadata from the result set
+        ResultSetMetaData rsmd = null;
+
+        // Will hold the number of columns in the result set
+        int columnCount = 0;
+
+        // Create a list to later hold the column names from metadata
+        List<String> columnNames = new ArrayList<String>();
+
+        // Load metadata from the result set and get column count while we're at it
+        try {
+            rsmd = r.getMetaData();
+            columnCount = rsmd.getColumnCount();
+        } catch (SQLException e) {
+            System.out.println("Error getting metadata");
+            e.printStackTrace();
+            return;
+        }
+
+        // Read the column names
+        // Do not modify the loop values: the result sets column index from 1 while a List index starts at 0.  The plus ones are to compensate for this.
+        for (int i = 0; i < columnCount; i++) {
+            try {
+                columnNames.add(rsmd.getColumnName(i + 1));
+            } catch (SQLException e) {
+                System.out.println("Error getting column name");
+                e.printStackTrace();
+                return;
+            }
+        }
```

Comment:
https://stackoverflow.com/a/10213258/14839305 Maybe consider something like this. May not be a good idea without making objects and may be over-engineered for our purposes

Reply:
Not very concerned with efficiency for the scope of this project, just need to resolve issues displaying the table in a properly formatted way for all cases
![image](https://user-images.githubusercontent.com/72429964/220228899-a485ecca-c523-4f98-a054-e962788fed65.png)